bot/seasons/evergreen/issues.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Issues(commands.Cog):
+    """Cog that allows users to retrieve issues from GitHub."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=("issues",))
+    async def issue(self, ctx, number: int, repository: str = "seasonalbot", user: str = "python-discord"):
+        """Command to retrieve issues from a GitHub repository."""
+        url = f"https://api.github.com/repos/{user}/{repository}/issues/{str(number)}"
+        status = {"404": f"Issue #{str(number)} doesn't exist in the repository {user}/{repository}.",
+                  "403": f"Rate limit exceeded. Please wait a while before trying again!"}
+
+        async with self.bot.http_session.get(url) as r:
+            json_data = await r.json()
+
+        if str(r.status) in status:
+            return await ctx.send(status.get(str(r.status)))
+
+        valid = discord.Embed(colour=0x00ff37)
+        valid.add_field(name="Repository", value=f"{user}/{repository}", inline=False)
+        valid.add_field(name="Issue Number", value=f"#{str(number)}", inline=False)
```

Comment:
String conversion is redundant in this case.

```suggestion
        valid.add_field(name="Issue Number", value=f"#{number}", inline=False)
```